server/src/__tests__/__snapshots__/server.test.ts.snap

```diff
@@ -653,11 +653,11 @@ exports[`server onRenameRequest Workspace-wide rename returns correct WorkspaceE
         "newText": "newName",
         "range": {
           "end": {
-            "character": 53,
+            "character": 10,
```

Comment:
Hm, why did these change? 🤔

Reply:
It’s because i moved tagRelease to a separate line, but removed one line above it (the comment) so the line number stays the same

Reply:
Ah, pretty nasty that I use the release script as a test file. My bad.

Reply:
No worries, it's not really supposed to change that often haha